schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/layout/config-editor.xml

```diff
@@ -77,14 +77,20 @@
     <for name="gco:Integer" use="number"/>
     <for name="gco:Real" use="number"/>
     <for name="gco:Angle" use="number"/>
+    <for name="gco:Measure" use="number"/>
+    <for name="gco:Length" use="number"/>
     <for name="gco:Boolean" use="checkbox"/>
-
+    <for name="gco:Measure/@uom" use="data-gn-keyword-picker">
+      <directiveAttributes
+        data-thesaurus-key="xpath::../*:dimensionName/*[@codeListValue]/(if (@codeListValue = 'time') then 'local.theme.codelist_unit_time' else 'local.theme.codelist_unit_distance')"
```

Comment:
If thesaurus is not available, there is no unit suggested.
But indeed, it could maybe make sense to create a better default view with this kind of field fully configured?

Reply:
Thesaurus added.